snmp/metadata.csv

```diff
@@ -1,2 +1,167 @@
+snmp.ipSystemStatsOutDiscards,count,,datagram,,"[Generic router] The number of output IP datagrams for which no problem was encountered to prevent their transmission to their destination, but were discarded due to other reasons, for example lack of buffer space.",0,snmp,snmp_snmp.ipSystemStatsOutDiscards
+snmp.ipSystemStatsOutFragCreates,count,,datagram,,[Generic router] The number of output datagram fragments generated as a result of IP fragmentation.,0,snmp,snmp_snmp.ipSystemStatsOutFragCreates
+snmp.ipSystemStatsOutFragFails,count,,datagram,,[Generic router] The number of IP datagrams discarded because they needed to be fragmented but could not be.,0,snmp,snmp_snmp.ipSystemStatsOutFragFails
+snmp.ipSystemStatsOutFragOKs,count,,datagram,,[Generic router] The number of IP datagrams successfully fragmented.,0,snmp,snmp_snmp.ipSystemStatsOutFragOKs
+snmp.ipSystemStatsOutFragReqds,count,,datagram,,[Generic router] The number of IP datagrams that would require fragmentation to be transmitted.,0,snmp,snmp_snmp.ipSystemStatsOutFragReqds
+snmp.ipSystemStatsOutNoRoutes,count,,datagram,,[Generic router] The number of locally generated IP datagrams discarded because no route could be found to transmit them to their destination.,0,snmp,snmp_snmp.ipSystemStatsOutNoRoutes
+snmp.ipSystemStatsReasmFails,count,,,,"[Generic router] The number of failures detected by the IP reassembly algorithm (for whatever reason: timed out, errors, etc.).",0,snmp,snmp_snmp.ipSystemStatsReasmFails
+snmp.ipSystemStatsReasmOKs,count,,datagram,,[Generic router] The number of IP datagrams successfully reassembled.,0,snmp,snmp_snmp.ipSystemStatsReasmOKs
+snmp.ipSystemStatsReasmReqds,count,,,,[Generic router] The number of IP fragments received that needed to be reassembled at this interface.,0,snmp,snmp_snmp.ipSystemStatsReasmReqds
+snmp.physicalDiskCapacityInMB,gauge,,mebibyte,,[Dell iDRAC] The size of the physical disk in megabytes.,0,snmp,snmp_snmp.physicalDiskCapacityInMB
+snmp.physicalDiskFreeSpaceInMB,gauge,,mebibyte,,[Dell iDRAC] The amount of free space in megabytes on the physical disk.,0,snmp,snmp_snmp.physicalDiskFreeSpaceInMB
+snmp.physicalDiskState,gauge,,,,"[Dell iDRAC] The current state of this physical disk. Possible states: 1 - The current state could not be determined., 2 - The physical disk is available for use, but no RAID configuration has been assigned. 3- A RAID configuration has been assigned to the physical disk. 4- The physical disk has been moved from another controller and contains all or some portion of a virtual disk. 5 - The physical disk is not available to the RAID controller. 6 - The physical disk is currently blocked by controller. 7 - The physical disk is not operational. 8 - The physical disk is not a RAID capable disk. 9 - The physical disk has been removed. 10 - The physical disk media has been placed in read only mode.",0,snmp,snmp_snmp.physicalDiskState
```

Comment:
This very long line (700+ characters) is causing the build to fail as it's over 400 characters. I'm not sure how well it would look on the UI anyway — do these states come from somewhere we could link to, perhaps?

Reply:
let me look if there is a doc out there about this.

Reply:
Found the link:

https://www.dell.com/support/manuals/us/en/04/dell-chassis-management-controller-v2.20-dell-poweredge-vrtx/snmp%20idracandcmc8.5/physical-disk-table?guid=guid-bbcc3e20-a879-40d2-94c8-05bbdc7bd086&lang=en-us

Reply:
Shortened: https://dtdg.co/snmp-physical-disk-state

Reply:
```suggestion
snmp.physicalDiskState,gauge,,,,"[Dell iDRAC] The current state of this physical disk. See Dell documentation to learn more: https://dtdg.co/snmp-physical-disk-state ",0,snmp,snmp_snmp.physicalDiskState
```

Reply:
What do you think about this ?